models/nc-news.model.js

```diff
@@ -43,9 +63,11 @@ const getArticlesCommentsCount = () => {
   articles.article_img_url,
   COUNT(comments) AS comment_count
   FROM articles JOIN comments ON articles.article_id = comments.article_id 
-  GROUP BY articles.article_id, articles.created_at
-  ORDER BY articles.created_at DESC;
-  `
+  ${topic ? "WHERE topic = $1" : ""}
```

Comment:
We need to do some kind of check to see if the provided topic is valid as right now it doesn't look like this is being checked anywhere?

Have another look at the notes on complex queries 👀 https://notes.northcoders.com/courses/js-back-end/complex-queries